docs/src/pages/index.mdx

```diff
@@ -14,3 +14,33 @@ This is `inlineCode`.
 ```
 Code block vanilla for now.
 ```
+
+```typescript
+import React from 'react'
+
+// comment here
+
+const Example: React.FC = () => {
+  return (
+    <div>
+      <p>hello world</p>
+    </div>
+  )
+}
+
+export default Example
+```
```

Comment:
a simple typescript codeblock. currently we don't show a preview but i expect that we will. also i haven't added the open in codesandbox integration because i wanted to try out the embedded codesandbox first...but i expect that we'll add codesandbox button and use their api to create the sandbox https://codesandbox.io/docs/importing